### ISSUE TITLE
Add me as a collaborator!

### ISSUE THREAD

**Issue by lukebelliveau** [Beacon24/v0#3]
**tl;dr**
You should add me as a collaborator to this repo so I can suggest changes. Do so like this: https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-personal-account-on-github/managing-access-to-your-personal-repositories/inviting-collaborators-to-a-personal-repository

**why?**
I have a suggested change, which I would normally submit via [pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-pull-requests). I do not have permissions to push branches to this repository, so I cannot open pull requests.

Once I am a collaborator, I can make Pull Requests. Pull Requests allow me to suggest changes to the code and provide a convenient interface for you to review these changes, make comments/suggestions, and approve/reject these changes.
